### DISCUSSION TITLE
Looking for Help & Community

### DISCUSSION THREAD

**Discussion by crackedbeefcake** [DioxusLabs/dioxus#1089] [General]
Hi,
I have set out to learn rust build an app in dioxus. The examples provided are all quite basic and don't tell much about how to structure larger applications in dioxus. 
Is there any community hubs? 
Also, i am sure this is not the right spot to place it but, I have opened a request for mentoring on codementor if anyone is interested.

**Comment by ealmloff** [comment]
You can join the Dioxus discord here: https://discord.gg/XgGxMSkvUM
That is the most active Dioxus community
